docs/application/dotnet/guides/nui/text.md

```diff
@@ -0,0 +1,590 @@
+## Text
+
+Therae are three Text components to display and edit text on the screen:
+
+- The [TextLabel](#textLabel) displays a short text string.
+
+- The [TextField](#textField) is to edit a single line text.
+
+- The [TextEditor](#textEditor) is to edit multi-line text.
+
+<a name="textLabel"></a>
+## TextLabel
+
+The `TextLabel` is a class that displays a short text string. `Tizen.NUI.BaseComponents.TextLabel` namespace contains the class.
+
+The `TextLabel` class is lightweight, non-editable, and do not respond to user input. Text labels support multiple languages and scripts including right-to-left scripts such as Arabic. For more information on how to display a text using a text label, see [NUI Hello World Tutorial](hello-world.md).

```

Comment:
Suggest the following change: 

Text labels are lightweight, non-editable, and do not respond to user input. Text labels support multiple languages and scripts including right-to-left scripts such as Arabic. For more information on how to display text using a text label, see [NUI Hello World Tutorial](hello-world.md).




To maintain consitency across Tizen documents, Refer :  
https://www.google.com/search?q=The+%60TextLabel%60+class+is+lightweight%2C+non-editable%2C+and+do+not+respond+to+user+input.&rlz=1C1CHBF_enIN851IN851&oq=The+%60TextLabel%60+class+is+lightweight%2C+non-editable%2C+and+do+not+respond+to+user+input.&aqs=chrome..69i57.1916j0j7&sourceid=chrome&ie=UTF-8